src/v/redpanda/application.cc

```diff
@@ -350,9 +379,26 @@ void application::wire_up_services() {
           _proxy,
           to_yaml(*_proxy_config),
           smp_service_groups.proxy_smp_sg(),
+          // TODO: Improve memory budget for services
+          // https://github.com/vectorizedio/redpanda/issues/1392
+          memory_groups::rpc_total_memory(),
           std::reference_wrapper(_proxy_client))
           .get();
     }
+    if (_schema_reg_config) {
+        construct_service(
+          _schema_registry_client, to_yaml(*_schema_reg_client_config))
+          .get();
+        construct_service(
+          _schema_registry,
+          to_yaml(*_schema_reg_config),
+          smp_service_groups.proxy_smp_sg(),
+          // TODO: Improve memory budget for services
+          // https://github.com/vectorizedio/redpanda/issues/1392
+          memory_groups::rpc_total_memory(),
```

Comment:
this makes a lot of sense. Since this is an external service, i think it should be tied to the Kafka-API-Rpc total memory, not the `rpc` which is internal.